Podfile.lock

```diff
@@ -0,0 +1,10 @@
+PODS:
+  - yajl (2.0.4)
+
+DEPENDENCIES:
+  - yajl (~> 2.0.4)
```

Comment:
Does the version number (2.0.4) have to be repeated four times? Seems error prone to me. I don't know the first thing about CocoaPods though.